bool cordinator
bool pickUp
bool stack
int16 mode